pyrit/prompt_converter/azure_speech_text_to_audio_converter.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import logging
+import pathlib
+import os
+import uuid
+
+import azure.cognitiveservices.speech as speechsdk
+from pyrit.memory.memory_models import PromptDataType
+from pyrit.prompt_converter import PromptConverter
+from pyrit.common import default_values
+from pyrit.common.path import RESULTS_PATH
+
+logger = logging.getLogger(__name__)
+
+
+class AzureSpeechTextToAudioConverter(PromptConverter):
+    """
+    The TextToAudio takes a prompt and generates a
+    wave file.
+
+    Args:
+        azure_speech_region (str): The name of the Azure region.
+        azure_speech_key (str): The API key for accessing the service.
+        synthesis_language (str): Synthesis voice language
+        synthesis_voice_name (str): Synthesis voice name, see URL
+        https://learn.microsoft.com/en-us/azure/ai-services/speech-service/language-support
```

Comment:
```suggestion
        synthesis_voice_name (str): Synthesis voice name, see URL
            https://learn.microsoft.com/en-us/azure/ai-services/speech-service/language-support
```
nit: indentation for line continuations is required if we ever format this into an API reference page

Reply:
fixed